doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='business-central-settings-changing-permissions-proc_{context}']
+= Changing permissions for groups and roles in {CENTRAL}
+
+In {CENTRAL}, you cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
```

Comment:
The changed permissions apply to users with the role or that belong to a group that you changed.

Reply:
Done